pkg/operator/ceph/object/spec.go

```diff
@@ -155,6 +155,21 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 		ServiceAccountName: serviceAccountName,
 	}
 
+	// If the rgw ops logs are enabled we add the side-car container
+	if c.store.Spec.RGWOpsLogEnabled {
+
+		monStore := cephconfig.GetMonStore(c.context, c.clusterInfo)
+		if err := monStore.Set("client.rgw", "rgw_enable_ops_log", "true"); err != nil {
```

Comment:
Does `ceph config set client.rgw <key> <val>` work? Will this apply the setting to all CephObjectStores? Will it apply it to none of the object stores? 

At any rate, I think this should be the client for only this objectstore.

Reply:
i was on the fence about this one, should we be enabling this setting from rook or let it be a manual operation, i will keep it for now though

Reply:
I think that's fine. It seems most important to me that Rook properly separate configs for each CephObjectStore.